js/app.js

```diff
@@ -0,0 +1,170 @@
+//Step 1 Declare Variables
+//Set Questions
+var questions = [{
+    //Question 1
+    question: "If you were strolling along this street, where would you be?",
```

Comment:
Is it considered bad practice to use single quotes for referencing files or elements and double quotes for strings? Would you reco just using one or the other for everything?